### PR TITLE
Milestone 3: Blog App: validations and model specs 

### PR DESCRIPTION
**Pull request by Gardimy**
In this project, I add validations to my models and create specs for them.

### CODE REVIEW

File: spec/models/post_spec.rb [hunk 1]
```diff
@@ -1,43 +1,43 @@
...
-#    end
-#  end
-#end
+      expect do
+        post.save
+        user.reload
+      end.to change(user, :posts_counter).by(1)
+    end
+  end
+end
```
Comment (tresorsawasawa) [inline spec/models/post_spec.rb]: - [ ] As required for the project to have tests to any custom methods in the models, kindly ensure that you also implement tests to the `#recent_comments` method in the `Post` model.

**Review by tresorsawasawa — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @Gardimy 
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

almost-there

## To highlight :champagne: 
- All tests are passing :heavy_check_mark: 
- All validations are correct :heavy_check_mark: 

## Required changes :recycle: :recycle: 
- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: spec/models/post_spec.rb (hunk 1 above)
Comment (Gardimy) [reply spec/models/post_spec.rb]: I had already completed it before I repeated the module. Once my previous project is approved I have only to add tests for the models. Do you want to check the project requirement?

Comment (tresorsawasawa) [reply spec/models/post_spec.rb]: @Gardimy 

But you didn't add tests for that method. Just add them and everything will be ok

**Review by codepantha — COMMENTED** [review]
## Hi @Gardimy 👋, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project that doesn't have any new commit after the previous reviewer's suggested changes 👍 



Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: spec/models/post_spec.rb (hunk 1 above)
Comment (Gardimy) [reply spec/models/post_spec.rb]: But I don't use this method in my post model.

Comment (tresorsawasawa) [reply spec/models/post_spec.rb]: @Gardimy 

Just check all methods you have in the Post model, and ensure that you have tests to all of them.

Comment (Gardimy) [reply spec/models/post_spec.rb]: ok!

**Review by Alfredbis29 — APPROVED** [review]
# STATUS: APPROVED   
 ## Hi @Gardimy 
Your project is complete! There is nothing else to say other than... Great job clap Congratulations! :tada:
 
 TO HIGHLIGHT:
 
 * Your project is professional :white_check_mark:
 * Good Job :+1:
 
 Now you can merge this branch and move on 1st_place_medal: HAPPY CODING AND GOOD LUCK WITH THE COMING ONES :crossed_fingers: :


 
 ## Optional suggestions

- [x] N/A

 _Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
 
Cheers and Happy coding! :clap:  :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me @Alfredbis29 in your question so I can receive the notification.
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**